net.waterfox.waterfox.yaml

```diff
@@ -0,0 +1,40 @@
+app-id: net.waterfox.waterfox
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: waterfox
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--filesystem=xdg-download:rw"
+  - "--persist=.waterfox"
+  - "--share=network"
+  - "--socket=pulseaudio"
+  - "--device=all"
+  - "--talk-name=org.freedesktop.Notifications"
+  - "--talk-name=org.freedesktop.FileManager1"
+  - "--talk-name=org.freedesktop.ScreenSaver"
+modules:
+  - shared-modules/dbus-glib/dbus-glib.json
+  - name: waterfox
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/opt
+      - cp -r waterfox/ /app/opt/waterfox
+      - desktop-file-install --dir /app/share/applications net.waterfox.waterfox.desktop
+      - install -Dm644 /app/opt/waterfox/browser/chrome/icons/default/default256.png /app/share/icons/hicolor/256x256/apps/net.waterfox.waterfox.png
+      - install -Dm644 net.waterfox.waterfox.appdata.xml -t /app/share/appdata
+      - install -Dm644 flatpak-prefs.js -t /app/opt/waterfox/browser/defaults/preferences/
+      - ln -sf /app/opt/waterfox/waterfox-bin /app/bin/waterfox
+    sources:
+      - type: archive
```

Comment:
Can you configure external-data-checker for that module? https://github.com/flathub/flatpak-external-data-checker/